### PR TITLE
Add v3 3.8b card

### PR DESCRIPTION
**Pull request by odashi**
v3 3.8Bモデル（約3.782B params）の仕様です。1.7Bと13Bの間を埋める仕様にしています。

### CODE REVIEW

File: models/v3-3.8b-exp1.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+|Initial learning rate|3e-4|
+|Final learning rate|3e-5|
+|Beta1|0.9|
+|Beta2|0.95|
+|Epsilon|1e-8|
+|Weight decay factor|0.1|
+|Warmup strategy|Linear|
+|Warmup steps|2000|
+|Learning rate scheduling strategy|Cosine|
+|Learning rate scheduling steps|492120|
```
Comment (Taishi-N324) [inline models/v3-3.8b-exp1.md:38]: `Learning rate scheduling steps` と `Total training steps` 数が異なるのは記入ミスでしょうか？

Comment (odashi) [reply models/v3-3.8b-exp1.md:38]: warmupが2000 stepsあるので、その差分でLR schedulingは2000少ない値になっています。

**Review by jnszk — APPROVED** [review]
